third_party/2and3/cryptography/hazmat/backends/interfaces.pyi

```diff
@@ -0,0 +1,13 @@
+class CMACBackend: ...
```

Comment:
Presumably these things have some methods? If you don't have time to stub them all out, we should add a `def __getattr__(self, name: str) -> Any` to avoid false positives for people using these classes.

Reply:
The backends themselves are private API.. so it's debatable whether they should be declared at all.

Reply:
They are [documented](https://cryptography.io/en/latest/hazmat/backends/interfaces/), though? Also, they should all have a `metaclass=ABCMeta` base. Same is true for several of the interfaces in other files.

Reply:
Would you have an example of this?